webextension/selector/ui.js

```diff
@@ -42,20 +42,13 @@ window.ui = (function () { // eslint-disable-line no-unused-vars
     return false;
   }
 
-  exports.isHeader = isHeader;
-
-  function htmlQuote(s) {
-    s = s + "";
-    return s.replace(/&/g, '&amp;').replace(/"/g, '&quot;').replace(/</g, '&lt;');
-  }
-
   let substitutedCss = inlineSelectionCss.replace(/MOZ_EXTENSION([^\"]+)/g, (match, filename) => {
     return chrome.extension.getURL(filename);
   });
 
   function makeEl(tagName, className) {
     if (! iframe.document) {
-      console.trace();
+      throw new Error("Attempted makeEl before iframe was initialized");
```

Comment:
It shouldn't happen, and I don't believe there are currently races, but it's an error when it does happen.  I believe we wait for initialization somewhere in uicontrol.